build.sbt

```diff
@@ -27,55 +20,63 @@ lazy val defaultSettings = Seq(
     collectionCompat,
     scalaOrganization.value % "scala-reflect" % scalaVersion.value % Provided
   )
-) ++ macros ++ simulacrumOptions ++ publishSettings
+) ++ macros ++ simulacrumOptions
 
-lazy val higherKindCore = project settings (
-  defaultSettings,
-  publishName := "core-higher-kind",
-  libraryDependencies ++= Seq(catsCore, catsFree, catsTagless),
-)
+lazy val higherKindCore = project
+  .in(file("modules/higherKindCore"))
+  .settings(
+    defaultSettings,
```

Comment:
each module should be provided with publish/noPublish settings

Reply:
I see, only docs don't get published. and what about the root itself?

Reply:
We are usually publishing it under `tofu`, it's like swissknife